### PR TITLE
Updated dark theme colors.

### PR DESCRIPTION
**Pull request by Vinicius149**
Changed the colors to blue.

### CODE REVIEW

File: desktop/src/main/java/haveno/desktop/theme-dark.css [hunk 1]
```diff
@@ -9,17 +9,17 @@
...
 
     /* javafx elements */
-    -fx-accent: #25b135;
+    -fx-accent: #0b65da;
     -fx-box-border: transparent;
-    -fx-focus-color: #2ea33c;
-    -fx-faint-focus-color: #2ea33c;
-    -fx-selection-bar: #1a6e1e;
-    -fx-selection-bar-non-focused: #2a7e2e;
+    -fx-focus-color: -fx-accent;
```
Comment (woodser) [inline desktop/src/main/java/haveno/desktop/theme-dark.css:14]: Thanks for your contribution. :) Overall it works.

I would expect consistency with Bisq's highlighting scheme.

In this case, focus-color, faint-focus-color, and color-primary-dark would have the same color.

And selection-bar and selection-bar-non-focused would have different colors.

Is the highlighting scheme changed intentionally, or should this be adjusted?

Were the highlighting

**Comment by woodser** [comment]
Ping @Vinicius149, I can accept as-is for the upcoming deployment unless you are taking a look per [my comment](https://github.com/haveno-dex/haveno/pull/604#discussion_r1176776524).

**Comment by woodser** [comment]
I've merged as-is, but feel free to update if you're inclined. Thanks :)

**Comment by Vinicius149** [comment]
I'm sorry for the absence, taking a little vacation from the job/studies. On the next upcoming days i will try to improve the changes, as said before, this was my first contribution to a project ever.

As soon i made the improvements a new PR will be submitted.

**Pull request merged**